packages/shared/lib/core/network/constants/destination-network-address.constant.ts

```diff
@@ -0,0 +1,6 @@
+import { DestinationNetwork } from '../enums'
+
+export const DESTINATION_NETWORK_ADDRESS: Readonly<{ [key in DestinationNetwork]?: string }> = {
+    [DestinationNetwork.Shimmer]: '-',
+    [DestinationNetwork.ShimmerEvm]: 'rms1qrut5ajyfrtgjs325kd9chwfwyyy2z3fewy4vgy0vvdtf2pr8prg5u3zwjn',
```

Comment:
Probably would be better to have a `const` object with both destination network addresses for testnet EVM _and_ ShimmerEVM.